tests/server/middleware.test.js

```diff
@@ -1,23 +1,40 @@
-const supertest = require("supertest");
+/**
+ * @jest-environment node
+ */
```

Comment:
Is this required or was it automatically added?

Reply:
required. jest throws some internal errors if we don't tell it this file should be tested in a node enviroment. not sure why the others ones are fine